### PR TITLE
Carousel

### PR DESCRIPTION
**Pull request by Fx-Colliot**

### CODE REVIEW

File: src/components/Logo.jsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+          customTransition="all .5"
+          transitionDuration={4000}
+          containerClass="carousel-container"
+          removeArrowOnDeviceType={["tablet", "mobile", "desktop"]}
+          dotListClass="custom-dot-list-style"
+          slidesToSlide="4"
+        >
+          {logos.map((logo) => {
+            return (
+              <Testing>
```
Comment (Thoms-C) [inline src/components/Logo.jsx]: We sould change the name of the styled-components

File: src/styled-components/FlexProjet.jsx [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+export const Center = styled.ul`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  @media ${Device.mobile} {
+    margin-left: 10vw;
+  }
+`;
+
+export const Tools = styled.li``;
```
Comment (Thoms-C) [inline src/styled-components/FlexProjet.jsx]: dead code

File: src/styled-components/FlexProjet.jsx [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+    flex-direction: column;
+    align-items: flex-start;
+    background-color: #0d253f;
+    color: white;
+    height: auto;
+    width: auto;
+  } ;
+`;
+
+export const Test = styled.ul`
```
Comment (Thoms-C) [inline src/styled-components/FlexProjet.jsx]: Rename this styled-components

File: src/styled-components/header/HeaderBodyStyled.jsx [hunk 4]
```diff
@@ -2,9 +2,14 @@ import styled from "styled-components";
 import { Device } from "../../assets/Device/Device.jsx";
 
 export const Page = styled.div`
+<<<<<<< HEAD
```
Comment (Thoms-C) [inline src/styled-components/header/HeaderBodyStyled.jsx]: Delete text from merging

**Review by Thoms-C — DISMISSED** [review]
Seems ok to me !

**Review by Thoms-C — DISMISSED** [review]
Ok!

File: debug.log [hunk 5]
```diff
@@ -0,0 +1 @@
+[0126/094543.885:ERROR:directory_reader_win.cc(43)] FindFirstFile: Le chemin d�acc�s sp�cifi� est introuvable. (0x3)
```
Comment (LoicBrassart) [inline debug.log]: What's this file ?

File: src/App.js [hunk 6]
```diff
@@ -65,6 +65,7 @@ export default function App() {
 
         <Route exact path="/mentionslegales" component={MentionsLegales} />
         <Route exact path="/contact" component={ContactMail} />
+
```
Comment (LoicBrassart) [inline src/App.js]: Why empty lines ?

File: src/components/Logo.jsx [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+    slidesToSlide: 1,
+  },
+};
+
+export default function Logo() {
+  const [logos, setLogos] = useState([]);
+
+  useEffect(() => {
+    axios
+      .get("http://localhost:5050/logos/")
```
Comment (LoicBrassart) [inline src/components/Logo.jsx]: localhost -> bad idea

File: src/components/ProjetsCards.jsx [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+  H3,
+} from "../styled-components/ProjetsCardsStyle";
+
+export default function ProjetsCards() {
+  const [projets, setProjets] = useState([]);
+  const [isDisplay, setDisplay] = useState({ isDisplay: false, id: "" });
+
+  useEffect(() => {
+    axios
+      .get("http://localhost:5050/projets")
```
Comment (LoicBrassart) [inline src/components/ProjetsCards.jsx]: Bad idea

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Minor evils

**Comment by LoicBrassart** [comment]
Well tried @Gauthier-Lhomme , but you still need to fix these evils :p

**Review by Thoms-C — DISMISSED** [review]
No more evils

File: src/components/Logo.jsx [hunk 9]
```diff
@@ -33,7 +34,7 @@ export default function Logo() {
 
   useEffect(() => {
     axios
-      .get("http://localhost:5050/logos/")
+      .get(`${api.defaults.baseURL}/logos/`)
```
Comment (LoicBrassart) [inline src/components/Logo.jsx]: I may be wrong, but this syntax is lighter and should work:
```
axios.get("/logos/")
```

File: src/components/ProjetsCards.jsx [hunk 10]
```diff
@@ -20,7 +22,7 @@ export default function ProjetsCards() {
 
   useEffect(() => {
     axios
-      .get("http://localhost:5050/projets")
+      .get(`${api.defaults.baseURL}/projets`)
```
Comment (LoicBrassart) [inline src/components/ProjetsCards.jsx]: I may be wrong, but this syntax is lighter and should work:
```
axios.get("/logos/")
```I may be wrong, but this syntax is lighter and should work:
```
axios.get("/logos/")
```

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Minor syntax Evil

**Review by Marie-Garde — APPROVED** [review]
ok

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**